test/can-view-parser-test.js

```diff
@@ -695,3 +695,44 @@ test('camelCase properties are encoded with on:, :to, :from, :bind bindings', fu
 
 	parser("<h1 on:aB='c' dE:to='f' gH:from='i' jK:bind='l'></h1>", makeChecks(tests));
 });
+
+//!steal-remove-start
+test('Warn on missing attribute value end quotes (canjs/can-view-parser#7)', function () {
```

Comment:
Can you update this to use `can-test-helpers` and the `testHelpers.dev.devOnlyTest` function instead of `steal-remove`ing the test. You can take a look at https://github.com/canjs/can-stache/blob/675fa5c3e9f09023b00566657bfce973654ec89c/test/stache-test.js#L200-L215 for an example.

Reply:
Done!